delfin/tests/unit/drivers/dell_emc/unity/test_emc_unity.py

```diff
@@ -421,7 +421,7 @@ def __init__(self):
     'status': 'abnormal',
     'name': 'CETV3182000026',
     'model': 'Unity 350F',
-    'raw_capacity': 2311766147072,
+    'raw_capacity': 8838774259712,
```

Comment:
Why this value need to be changed?

Reply:
because we use disk capacity as raw_capacity,it's incorrect，now use total_capacity,the value has changed